tests/test_hydraulic_power.py

```diff
@@ -367,6 +364,12 @@ def energy_system_options(self):
         np.testing.assert_allclose(pipe_hp[1], a[1] * results[f"{pipe}.GasOut.Q"][1] + b[1])
         np.testing.assert_allclose(pipe_hp[2], a[2] * results[f"{pipe}.GasOut.Q"][2] + b[2])
 
+        # gas demand matching
+        np.testing.assert_allclose(
```

Comment:
this demand matching is in the demand_matching_test from utils_test.py.
I prefer to use that one, as it is more generic.

Reply:
Previously this did not work for gas. Thanks for making me aware that it works now. It has been updated accordingly